client/src/pages/dashboard/Dashboard.js

```diff
@@ -77,13 +90,29 @@ function Dashboard() {
 
   return (
     <div className="dashboard">
-      <div className="navbar">this is the delicious navbar</div>
       <div className="dashboard-screen">
         <div className="proposal-list">
           <SearchBar keyword={input} setKeyword={updateInput}/>
           {proposalList}
         </div>
-        <div className="proposal-description"></div>
+        <div className="proposal-description">
+          <div className="proposal-background">
+            <div className="proposal-head-title">{proposalTitle}</div>
+            <div className="proposal-head-description">{proposalDescription}</div>
+            <div className="dividing-line"> </div>
+            <div className="comment-area">
+              <textarea id="textbox" className="comment-box" rows="7" cols="55"> Please enter text here... </textarea>
```

Comment:
Use the "placeholder" attribute of textarea for nice placeholder text so you don't have to manually delete the default text